File: app/src/main/java/closer/vlllage/com/closer/handler/helpers/ViewAttributeHandler.java
package closer.vlllage.com.closer.handler.helpers;

import android.view.View;

import closer.vlllage.com.closer.pool.PoolMember;

public class ViewAttributeHandler extends PoolMember {
    public void linkPadding(View targetView, View sourceView) {
        sourceView.addOnLayoutChangeListener((v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom) -> {
            targetView.setPadding(sourceView.getPaddingLeft(), sourceView.getPaddingTop(),
                    sourceView.getPaddingRight(), sourceView.getPaddingBottom());
        });
    }
}

File: app/src/main/java/closer/vlllage/com/closer/handler/helpers/ViewAttributeHandler.kt
package closer.vlllage.com.closer.handler.helpers

import android.view.View

import com.queatz.on.On

class ViewAttributeHandler constructor(private val on: On) {
    fun linkPadding(targetView: View, sourceView: View) {
        sourceView.addOnLayoutChangeListener { v, left, top, right, bottom, oldLeft, oldTop, oldRight, oldBottom ->
            targetView.setPadding(sourceView.paddingLeft, sourceView.paddingTop,
                    sourceView.paddingRight, sourceView.paddingBottom)
        }
    }
}
